src/modules/common/reducer.ts

```diff
@@ -57,6 +58,7 @@ export function createRootReducer(history: History) {
       item,
       collection,
       location,
+      stats,
```

Comment:
Couldn't `stats` be a bit too generic? or are we planning to re-use it if we ever need to display stats elsewhere?

Also! maybe it's a bit weird that the types and part of the code call the Stats `WeeklyStats` but the store/variable names don't reflect that

Reply:
Maybe, what name would you recommend? I didn't use analytics because that's an already existing module for our analytics.

Regarding thhe second point, there was a discussion to add daily/monthly analytics too, and being able to switch from a dropdown, but it required some work on the redshift side. That's why i named the endpoint and the data type "weekly", I can wrap these stats under `data.weekly` instead of just `data`, would that make it clearer?

Reply:
> Maybe, what name would you recommend? I didn't use analytics because that's an already existing module for our analytics.

I'm not sureeeee. Will ping you on Slack about it 🚀

> Regarding thhe second point, there was a discussion to add daily/monthly analytics too, and being able to switch from a dropdown, but it required some work on the redshift side. That's why i named the endpoint and the data type "weekly", I can wrap these stats under data.weekly instead of just data, would that make it clearer?

Oh! maybe it's not necessary then, we can do that when we get there. If you think it makes it a bit clearer in case someone else has to develop that feature go right ahead! we can also leave a comment

Reply:
i added "weekly" to the actions names and within the reducer data